packages/utils/src/lib/network.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Just a wrapper around `window.fetch` that sets the `referrerPolicy` to `strict-origin-when-cross-origin`.
+ *
+ * @public
```

Comment:
let's mark this as `@internal` - i don't know if we want it as part of our public API